src/env/env.c

```diff
@@ -0,0 +1,94 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   env.c                                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: alopez-g <marvin@42.fr>                    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/10/15 18:07:51 by alopez-g          #+#    #+#             */
+/*   Updated: 2022/10/16 01:18:23 by alopez-g         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "env.h"
+
+//easy version
+void	env_unsetvar(char ***env, char *key)
+{
+	int		i;
+	int		esize;
+
+	i = 0;
+	esize = env_size(*env);
+	while (*(*env + i))
+	{
+		if (key_compare(*(*env + i), key))
+		{
+			free(*(*env + i));
+			*(*env + i) = *(*env + esize - 1);
```

Comment:
cool :)
you can create a swap_func() if you want

Reply:
well i made a fill_mem func, since it is not exactly a swap, it is more like freeing memory but filling that empty space